Lesson07

pkoleyni opened the pull request.

pkoleyni committed:
Lesson07

ldconejo commented:
Hi Pooria,

Code looks good, a few things to look at and correct:

1. Please add some notes to _findings.txt_, explaining the reason for the execution time improvement. Also, note that line 9 says "Running parallel.py takes 1.6s". I think this one really refers to _serial.py_.

2. _parallel.py_ reports 0 records added for _product_info_ and _rental_info_. Based on your input files, they should be reporting 2 and 6, respectively.

```
$ python parallel.py 
(80, 403, 483, 0.5125241279602051)
(0, 0, 14, 0.513098955154419)
(0, 303, 363, 0.5769531726837158)
Total time: 0.5791871547698975
```

Regards,

Luis

pkoleyni commented:
Hi Luis
Thanks for the feedback, i have fixed parallel.py code and added more
context and info to my findings
Would you please review and let me know if i need to change anything to get
full grade
Many thanks,
Pooria

On Thu, Nov 21, 2019 at 11:15 PM Luis Conejo-Alpizar <
notifications@github.com> wrote:

> Hi Pooria,
>
> Code looks good, a few things to look at and correct:
>
>    1.
>
>    Please add some notes to *findings.txt*, explaining the reason for the
>    execution time improvement. Also, note that line 9 says "Running
>    parallel.py takes 1.6s". I think this one really refers to *serial.py*.
>    2.
>
>    *parallel.py* reports 0 records added for *product_info* and
>    *rental_info*. Based on your input files, they should be reporting 2
>    and 6, respectively.
>
> $ python parallel.py
> (80, 403, 483, 0.5125241279602051)
> (0, 0, 14, 0.513098955154419)
> (0, 303, 363, 0.5769531726837158)
> Total time: 0.5791871547698975
>
> Regards,
>
> Luis
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/pull/330?email_source=notifications&email_token=AL4JFEBDMIEUVV7SZEP3FPDQU6BHTA5CNFSM4JQLDBV2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEE4Y4OQ#issuecomment-557420090>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4JFEDV6HZJZA3KNWCIIJ3QU6BHTANCNFSM4JQLDBVQ>
> .
>


-- 
---------------------------
*Pooria Koleyni*
www.pooriakoleyni.com

ldconejo commented:
Hi Pooria,

Looks good, thanks for taking the extra time to implement the feedback from the first review. Good luck on your next assignment!

Regards,

Luis

pkoleyni commented:
Thank you

On Sat, Nov 23, 2019 at 7:36 PM Luis Conejo-Alpizar <
notifications@github.com> wrote:

> Hi Pooria,
>
> Looks good, thanks for taking the extra time to implement the feedback
> from the first review. Good luck on your next assignment!
>
> Regards,
>
> Luis
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/pull/330?email_source=notifications&email_token=AL4JFEGXHYDZ7OEMQ3QE2K3QVHZDJA5CNFSM4JQLDBV2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFAC6QI#issuecomment-557854529>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL4JFEHPSNGMAYKHYP4K3LTQVHZDJANCNFSM4JQLDBVQ>
> .
>
-- 
---------------------------
*Pooria Koleyni*
www.pooriakoleyni.com
